db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 import_fixture 'practices'
 import_fixture 'categories'
 import_fixture 'learnings'
+import_fixture "reports"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.